docs/gatsby-config.mjs

```diff
@@ -54,22 +54,28 @@ const config = {
 	plugins: [
 		`gatsby-plugin-sharp`,
 		`gatsby-transformer-sharp`,
-		{
-			resolve: 'gatsby-plugin-react-svg',
-			options: {
-				rule: {
-					include: '/icons/',
-				},
-			},
-		},
-		'gatsby-transformer-yaml',
+		// {
```

Comment:
wanna remove this?

Reply:
They have been removed from both the config and the packages list.